src/core/async/modules/wrappers/README.md

```diff
@@ -0,0 +1,39 @@
+# core/async/modules/wrappers
+
+This module provides Async wrappers for data-providers and event-emitters.
+
+### Data-providers
+
```

Comment:
The wrapper takes a link to the "raw" data provider and returns a new object that based on the original, but all async methods and properties are wrapped by Async.